src/App.js

```diff
@@ -1,8 +1,65 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉